WalletWasabi.Fluent/ViewModels/AddWallet/Create/ConfirmRecoveryWordsViewModel.cs

```diff
@@ -50,8 +50,15 @@ public ConfirmRecoveryWordsViewModel(List<RecoveryWordViewModel> mnemonicWords,
 				.Bind(out _confirmationWords)
 				.Subscribe();
 
+#if DEBUG
+			var verificationWordsCount = 3;
+#else
+			var verificationWordsCount = 12;
+#endif
+
```

Comment:
Sorry @jmacato but historically we have been against preprocessor directives. Couldn't you infer the final value based on the network or any other way?

Reply:
Just curious why are you against that?

Reply:
Hmm.. it'd make sense to just enforce this on Mainnet i guess, is that alright?

Reply:
Since there is the skip button for the testnet and regtest, I think you dont need to enforce anything

Reply:
> Hmm.. it'd make sense to just enforce this on Mainnet i guess, is that alright?

Yes. You already have some `_isSkipEnable` variable, just in case.

> Just curious why are you against that?

Because you have two different binary/products. One that runs on debug and the real one that runs on release.